lib/vmq_elixir_plugin.ex

```diff
@@ -22,17 +20,7 @@ defmodule VmqElixirPlugin do
   Application start function.
   """
   def start(_type, _args) do
-    import Supervisor.Spec, warn: false
-
-    children = [
-      # Define workers and child supervisors to be supervised
-      # worker(VmqElixirPlugin.Worker, [arg1, arg2, arg3])
-    ]
-
-    # See http://elixir-lang.org/docs/stable/elixir/Supervisor.html
-    # for other strategies and supported options
-    opts = [strategy: :one_for_one, name: VmqElixirPlugin.Supervisor]
-    Supervisor.start_link(children, opts)
+    :ok
```

Comment:
I think that's the spec for the `Application.start/2` function and no the spec for the application behaviour callback function. In `application.erl` the `start/2` application callback is defined:

```
-callback start(StartType :: start_type(), StartArgs :: term()) ->
    {'ok', pid()} | {'ok', pid(), State :: term()} | {'error', Reason :: term()}.
```
Anyway, we also don't need the `use Application` line at all if we're turning this into a library application.